backend/src/routes/notes.ts

```diff
@@ -34,15 +37,48 @@ interface NoteItem {
 ]
  */
 router.get("/notes/:id", async (req: Request, res: Response, next: NextFunction) => {
-  try {
-    console.log("Getting...");
+  try{
     const id = req.params.id;
     const note = await Note.findById(id);
     if (note == null) {
-      throw new Error();
+      return res.status(ServiceError.NOTE_WAS_NOT_FOUND.status)
+          .send(ServiceError.NOTE_WAS_NOT_FOUND.message);
     }
-    const notes: NoteItem[] = note.answers as NoteItem[];
-    notes.forEach((note_answer) => {
+    if (note.type==="post"){
```

Comment:
this might be a difference in version, but on main I believe we have "pre", "postMentor", and "postMentee" as a note type